Handdeck/Include/Handdeck.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include <vector>
+#include <string>
+#include <iostream>
+#include <string>
+#include <vector>
+#include <cstdlib>
+#include <ctime>
+
+
+using Card = std::string;
+
+class Deck
+{
+public:
+  std::vector<Card> Cards;
+  void addCard(Card cardToAdd);
+};
+
+class DiscardDeck :public Deck
+{
+public:
+  int maxSize;
+  DiscardDeck(int _maxSize);
+};
+
+class PullDeck :public Deck
+{
+public:
+  int maxSize;

```

Comment:
kolejny maxSize, który jest nie wiadomo po co? Skoro jest wszędzie to daje się go do klasy, z której dziedziczymy